packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/aspnet/integrationsteps.md

```diff
@@ -190,14 +190,14 @@ var enrollAuthenticatorOptions = new EnrollAuthenticatorOptions
 var enrollResponse = await idxAuthClient.EnrollAuthenticatorAsync(enrollAuthenticatorOptions, (IIdxContext)Session["IdxContext"]);
 ```
 
-### Step 12: Update the Email Factor Verification email template (optional)
+### Step 12 Optional: Update sent email template
```

Comment:
Put "Optional" in parens (i.e. `(Optional)`) for consistency with other guides.

Reply:
Also check other instances of "Optional"

Reply:
Added parentheses and also checked the other optionals I changed and ensured consistency.